modules/tests/jvm/src/main/scala/lucuma/catalog/AgsSelectionApp.scala

```diff
@@ -72,11 +76,17 @@ trait AgsSelectionSample {
   val offsets =
     NonEmptyList.of(Offset.Zero, Offset.Zero.copy(q = Offset.Q(Angle.fromDoubleArcseconds(15))))
 
-  val params = AgsParams.GmosAgsParams(
```

Comment:
I guess this was never used before?

Reply:
Right, it was a bit surprising :D

Reply:
Note from the future: It is (was) used in AgsBenchmark.scala. 😆 Apparently it is not compiled in CI.